Content.Server/_ES/Auditions/AuditionsSystem.cs

```diff
@@ -0,0 +1,59 @@
+using Content.Server.Maps;
+using Content.Shared._ES.Auditions;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server._ES.Auditions;
+
+/// <summary>
+/// This handles the server-side of auditioning!
+/// </summary>
+public sealed class AuditionsSystem : SharedAuditionsSystem
+{
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    /// <summary>
+    /// Hires a cast, and integrates relationships between all of the characters.
+    /// </summary>
+    /// <param name="captainCount"></param>
+    /// <param name="minimumCrew"></param>
+    /// <param name="maximumCrew"></param>
+    /// <param name="lowPopPoolPrototype"></param>
+    /// <param name="highPopPoolPrototype"></param>
+    /// <param name="producer"></param>
+    /// <exception cref="Exception"></exception>
+    public void GenerateCast(
+        int captainCount = 26,
+        int minimumCrew = 5,
+        int maximumCrew = 12,
+        [ForbidLiteral] string lowPopPoolPrototype = "DefaultShipPoolLowPop",
+        [ForbidLiteral] string highPopPoolPrototype = "DefaultShipPoolHighPop",
```

Comment:
i feel like im slightly hallucinating. neither of these prototypes exist and i dont see why you simultaneously need to pass in multiple game map pools and a crew range. I think it would be better if you just passed in a list of crew sizes that you want to generate for and it just divvies it up.

like if i want 30 crews and i pass in 5, 7, and 12, then it generate 10 of each.